src/main/java/org/abstractj/kalium/keys/SigningKey.java

```diff
@@ -60,6 +60,12 @@ public VerifyKey getVerifyKey() {
         return this.verifyKey;
     }
 
+    public PrivateKey getPrivateKey() {
+        byte []secretKey = zeros(CRYPTO_SCALARMULT_CURVE25519_BYTES);
+        sodium().crypto_sign_ed25519_sk_to_curve25519(secretKey, seed);
```

Comment:
Why you would like to expose the private key? Also, I suggest to change the method name to something like `convertToCurve`.

Just doing it by default feels wrong